Jenkinsfile

```diff
@@ -61,7 +56,8 @@ pipeline {
                   script {
                     def dockerRepoName = 'zooniverse/fe-content-pages'
                     def dockerImageName = "${dockerRepoName}:${GIT_COMMIT}"
-                    def newImage = docker.build(dockerImageName)
+                    def buildArgs = "--build-arg ASSET_PREFIX SENTRY_DSN ."
```

Comment:
Should this be `"--build-arg ASSET_PREFIX --build-arg SENTRY_DSN ./"` I've been reading the docs and advice for build args but I'm not sure what the syntax is.

Reply:
yep - they need to be specified for each env var. thanks i'll update